src/client/Microsoft.Identity.Client/AppConfig/ConfidentialClientApplicationBuilder.cs

```diff
@@ -399,6 +406,12 @@ internal override void Validate()
                     Config.AzureRegion = forcedRegion;
                 }
             }
+

```

Comment:
Why did you move this check after the if/else instead of keeping it before? Do you need to add `return`s to the if/else if?

Reply:
Fair question, I'll revert.